flash_image/index.rst

```diff
@@ -3,7 +3,13 @@
 How to flash image
 ##################
 
-TODO
+Prerequisite
```

Comment:
Don't make Prerequisite be the headline.

Reply:
OK.